agg-ast/schema_derivation/src/lib.rs

```diff
@@ -155,6 +155,49 @@ fn get_schema_for_path_mut_aux(
     }
 }
 
+/// Gets a mutable reference to a specific field or document path in the schema.
```

Comment:
Outdated comment? This is the one that does not return a mutable ref

Reply:
yup just copy and pasted the original func 🤦